src/components/Common/UserAutocompleteFormField.tsx

```diff
@@ -80,6 +80,7 @@ export default function UserAutocomplete(props: UserSearchProps) {
         )}
         optionLabel={formatName}
         optionIcon={userOnlineDot}
+        optionImage={(option) => option.read_profile_picture_url || ""}
```

Comment:
and hence you wouldn't need this hack in the first place

```suggestion
        optionImage={(option) => option.read_profile_picture_url}
```